src/app/routes/frontPage/getInitialData/index.js

```diff
@@ -20,9 +31,34 @@ const transformJson = pipe(
 const getRadioScheduleToggle = path(['frontPageRadioSchedule', 'enabled']);
 const getRadioSchedulePosition = path(['frontPageRadioSchedule', 'value']);
 
-export default async ({ path: pathname, service, pageType, toggles }) => {
+export default async ({
+  path: pathname,
+  service,
+  variant,
+  pageType,
+  toggles,
+}) => {
+  const env = getEnvironment(pathname);
+  const isLocal = !env || env === 'local';
+
+  const agent = !isLocal ? await getAgent() : null;
+
+  const fetchUrl = constructPageFetchUrl({
+    pathname,
+    pageType: CPS_ASSET,
```

Comment:
Can we not use the `FRONT_PAGE` value here?

Reply:
Oh... maybe we can... I thought that this pageType mapped directly to the BFF page type, and we don't have a pageType for FrontPage - it's just a CPS asset with a slightly different ID. 

Let me see if changing it to FRONT_PAGE might work - it might make some of the workarounds below a bit easier to understand.

Reply:
Hmm pageType is used to construct the queryParams: https://github.com/bbc/simorgh/blob/7801cabd3181299da3a230a240f54762ef3a6a65/src/app/routes/utils/constructPageFetchUrl/index.ts#L96-L107

I could do something (hacky) like have a derivedPageType / mappedPageType - if pageType is `FRONT_PAGE`, use `CPS_ASSET` perhaps?

Reply:
Ah I see. Maybe a comment on it for now is ok to explain why `CPS_ASSET` is being passed in? They'll be moving over to the new Homepages overtime won't they, so probs not a big deal.

Reply:
Yes, they will indeed be migrated eventually, so I can add a comment for now, and this will eventually be deleted once all FrontPages are migrated to HomePage